tests/e2e-openshift/otlp-metrics/04-metrics-traces-gen.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: otelgen-traces
+spec:
+  completions: 1
+  parallelism: 1
+  template:
+    metadata:
+      labels:
+        app: otelgen
+    spec:
+      containers:
+        - name: otelgen-traces-multi
+          image: ghcr.io/krzko/otelgen:latest
+          env:
+            - name: OTLP_ENDPOINT
+              value: cluster-collector-collector-headless.kuttl-otlp-metrics.svc:4317
+          command: ["./otelgen"]
+          args:
+            - "--otel-exporter-otlp-endpoint=$(OTLP_ENDPOINT)"
+            - "--log-level=debug"
+            - "--insecure=true"
+            - "--duration=30"
+            - "traces"
+            - "multi"
+      restartPolicy: Never
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: otelgen-metrics
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: otelgen
+  template:
+    metadata:
+      labels:
+        app: otelgen
+    spec:
+      containers:
+        - name: otelgen-metrics-counter
+          image: ghcr.io/krzko/otelgen:latest
```

Comment:
The same as before ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen

Reply:
used the offical tooling.